@types/mojave.d.ts

```diff
@@ -8,7 +9,7 @@ declare namespace mojave
         }
     };
     export type MountableType = "func" | "jsx" | "class";
-    export type MountableFunction = (element: HTMLElement, ...args: any[]) => void;
+    export type MountableFunction = (element: HTMLElement, ...args: any[]) => any;
```

Comment:
Why is that change necessary?

Reply:
It seemed unnecessary restrictive that we enforce the function to return `void`. In theory they could return whatever they want, it's just that `mount*` doesn't handle/return it.